war/src/main/webapp/scripts/sortable.js

```diff
@@ -266,6 +270,35 @@ var Sortable = (function() {
     arrowTable.down.next = arrowTable.up;
 
 
+    function alphanum_sort(a,b,caseInsensitive) {
+        function chunkify(t) {
+          var tz = [];
+          var x = 0, y = -1, n = 0, i, j;
+
+          while (i = (j = t.charAt(x++)).charCodeAt(0)) {
+            var m = (i == 46 || (i >=48 && i <= 57));
+            if (m !== n) {
+              tz[++y] = "";
+              n = m;
+            }
+            tz[y] += j;
+          }
+          return tz;
+        }
+
+        var aa = chunkify(caseInsensitive ? a.toLowerCase() : a);
+        var bb = chunkify(caseInsensitive ? b.toLowerCase() : b);
+
+        for (x = 0; aa[x] && bb[x]; x++) {
+          if (aa[x] !== bb[x]) {
+            var c = Number(aa[x]), d = Number(bb[x]);
+            if (c == aa[x] && d == bb[x]) {
+              return c - d;
+            } else return (aa[x] > bb[x]) ? 1 : -1;
+          }
+        }
+        return aa.length - bb.length;
+    }
```

Comment:
I'm the author of the Alphanum algorithm and I have no specific reason to maintain the LGPL license on it. I'd like to change Alphanum's licensing and I might as well select a license that will satisfy the use of the code in Jenkins. I imagine an MIT license would fit nicely with Jenkins. Would an Apache license also be acceptable?